src/components/SidePanel/NavGroup/FilterNavGroup/FilterGroup/index.tsx

```diff
@@ -62,7 +79,16 @@ const FilterGroup: React.FC<FilterGroupProps> = ({
   return (
     <FilterContainer data-cy={dataCy}>
       <FilterHeader>
-        <Badge>FILTER</Badge>
+        <BadgeWrapper>
+          <Badge>FILTER</Badge>
```

Comment:
[nit] The badge applies capitalization itself
```suggestion
          <Badge>Filter</Badge>
```

Reply:
Bump

Reply:
Bump